src/signalingclient/DefaultSignalingClient.ts

```diff
@@ -273,9 +273,12 @@ export default class DefaultSignalingClient implements SignalingClient {
       this.webSocket.addEventListener('close', handler);
       scheduler.start(() => {
         // SDK has not received the "close" event on WebSocket for two seconds.
-        // Handle a fake close event with 1006 to indicate that the client abnormally closed the connection.
+        //
+        // We report it as a graceful close with no message as there should be no end-user
+        // impact of this timeout, and we do not want to trigger any non-graceful disconnection handling
+        // like reconnection.
         handler(
-          new CloseEvent('close', { wasClean: false, code: 1006, reason: '', bubbles: false })
+          new CloseEvent('close', { wasClean: false, code: 1005, reason: '', bubbles: false })
```

Comment:
Another change brought in from previous commit to avoid headaches in the future.